app/scripts/modules/core/src/projects/project.less

```diff
@@ -57,6 +58,12 @@
       margin-top: -10px;
     }
   }
+
+  .insight {
+    .detail-content {
+      height: calc(~'100% - 54px');
```

Comment:
Is it worth storing 54px as a variable that's shared between here and the header height?

Reply:
👍